about/roles.md

```diff
@@ -150,25 +145,25 @@ bacon-makers!
 <h3 id="sponsors">Sponsors</h3>
 
 Sponsors make it possible for Rails Girls Summer of Code to create a paid
-program for participants. Sponsors are sourced from all over the world, and
-represent an entire cross section of industry. While Sponsors receive a great
-deal of publicity and public exposure in return for their generous donations
-(usually a package) but most of the time not being involved actively sponsors
-take on a supporting actors role, but quite obviously a very importing one:
-They essentially pay for students so they are able to participate in this
-program.
+program for participants. Sourced from all of the world, sponsors represent
+a wide cross section of industry and allow for students to participate in the 
+program by making generous donations. Though they often receive a great
+deal of publicity and exposure as a result, they are not often directly
+involved in the projects themselves. Instead they take on a more supporting
+role in the experience of participants. 
+
 
 <h3 id="coaching_company">Coaching Companies</h3>
 
-This role is a formalization of a very successful concept from last year:
-Companies that fully host a students team for the duration of their project and
-allow employees to coach them. You can find out more about this initiative
-here: <a href="/guide/coaching-company">Coaching Companies</a>. Sponsors can be coaching
-companies too, as well as vice versa
+A Coaching Company is one that host an entire student team for the duration of 
+their project. In addition, they can allow employees to coach the participants. 
```

Comment:
could you remove “can” here? 
So basically: “In addition, they allow employees to coach the participants”